docker-compose.yml

```diff
@@ -53,3 +54,25 @@ services:
       - env.d/development/crowdin
     user: "${DOCKER_USER:-1000}"
     working_dir: /app
+
+  redis-sentinel:
+    image: docker.io/bitnami/redis-sentinel:6.0-debian-10
+    depends_on:
+      - redis-primary
+      - redis-replica1
+      - redis-replica2
+
+  redis-primary:
+    image: docker.io/bitnami/redis:6.0-debian-10
```

Comment:
Because the bitnami image simplifies the setup of a redis cluster (with the environment variable `REDIS_REPLICATION_MODE` that I forgot to set here... :man_facepalming:  )

Reply:
ok then